src/App.js

```diff
@@ -1,24 +1,39 @@
-import logo from './logo.svg';
 import './App.css';
+import CountriesList from './components/CountriesList';
+import CountryDetails from './components/CountryDetails';
+import Navbar from './components/Navbar';
+import { Routes, Route } from 'react-router-dom';
+import { useEffect, useState} from 'react';
+import countriesArr from "./countries.json"
+import axios from 'axios';
 
 function App() {
+
+ 
+  const [countries, setCountries] = useState(countriesArr);
+
+ 
+  useEffect(() => {
+    axios.get("https://ih-countries-api.herokuapp.com/countries")
+    .then( response => {
+      setCountries(response.data); //save into state variable
+    })
+    .catch( e => console.log("error getting countriesfrom API", e))
+  }, [])
+
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Navbar/>
+      <CountriesList listOfCountries={countries}/>
+
```

Comment:
this is correct, but if you want you can change it to this:
```
<Routes>
      <Route path="/" element={ <CountriesList listOfCountries={countries}/> } />
      <Route path="/countries/:countryId" element={ <CountryDetails allCountries={countries} /> } />     
 </Routes>
```
because without styling you need to scroll to the end of the page to see country details